src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  padding: 10px;
+
+  display: flex;
+  flex-flow: row wrap;
+
+  gap: $gap-between-days;
+  width: ($amount-of-day * $day-size) + (($amount-of-day - 1) * $gap-between-days) + 20px;
+
+  &__day {
+    position: relative;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $day-size;
+    height: $day-size;
+    background-color: $day-color;
+
+    border: 1px solid $border-color;
+
+    transition: all 0.5s;
```

Comment:
Specify to which properties should transition apply

Reply:
@BudnikOleksii All changes should be animated with the duration of 0.5s

or no?

Reply:
> @BudnikOleksii All changes should be animated with the duration of 0.5s
> 
> or no?

It's better to avoid using `all` as it can lead to problems with preformance